src/excalidraw-app/components/ExportToExcalidrawPlus.tsx

```diff
@@ -0,0 +1,99 @@
+      fill="currentColor"
+    />
+  </>,
+  { width: 89, height: 131, style: { transform: "translateX(4px)" } },
+);
+
+const encryptData = async (
```

Comment:
yeah we should soon factor out encrypt helper and move to `excalidraw-app/data/index.ts`